tests/lib/fde-setup-hook/fde-setup.go

```diff
@@ -130,23 +155,37 @@ func runFdeRevealKey() error {
 		return fmt.Errorf("fde-reveal-key input is not valid base64: %v", err)
```

Comment:
"fde-reveal-key **key** input is not valida base64: %v" to follow the other error messages?